README.md

```diff
@@ -1,6 +1,6 @@
 # Moodle Enrolment key based self-registration
 
-![GitHub Workflow Status (branch)](https://img.shields.io/github/workflow/status/catalyst/moodle-auth_enrolkey/ci/master?label=ci)
+![GitHub Workflow Status (branch)](https://img.shields.io/github/actions/workflow/status/catalyst/moodle-auth_enrolkey/ci.yml?branch=master)
```

Comment:
this needs to point at the stable branch

Reply:
done